Modelica/Math/package.mo

```diff
@@ -8000,9 +8000,9 @@ For details of the arguments, see documentation of dgbsv.
 
       input Real A[:, size(A, 1)];
       input Integer ilo=1
-        "Lowest index where the original matrix had been Hessenberg form";
+        "Lowest index where the original matrix have not been in upper triangular form";
```

Comment:
have or has? I am not sure.

Reply:
Should likely be singular; but on second thought the original matrix is unchanged so I believe it is simpler to use "is" leading to "where the original matrix is not in upper triangular form".

Reply:
Shall I push the change to "is not in upper triangular form"?
Shall I also change this for ilo/ihi for dormhr and dorghr?
(As I understand they are all identical.)

Reply:
Yes, please.

Reply:
Done now.